meson_options.txt

```diff
@@ -0,0 +1,11 @@
+option('share-plugins',
+        type: 'string',
+        value: 'browser,email,instapaper,pocket,telegram,twitter,wallabag',
```

Comment:
As we have bumped the meson dependency, we can use the array type instead 
http://mesonbuild.com/Build-options.html#arrays